lib/mqtt/mqtt.cc

```diff
@@ -108,19 +107,12 @@ namespace
 	}
 
 	/**
-	 * Helper to determine if a passed TLS connection is terminated. This
-	 * is useful to diagnose coreMQTT failures and return a clear error
-	 * code to the caller.
+	 * Returns a weak pseudo-random number.
 	 */
-	bool is_tls_terminated(SObj tlsHandle)
+	uint64_t rand()
 	{
-		// If the link fails with -ENOTCONN as part of `transport_recv`
-		// or `transport_send`, the `tlsHandle` is invalidated.
-		if (!Capability{tlsHandle}.is_valid())
-		{
-			return true;
-		}
-		return false;
+		EntropySource rng;
```

Comment:
This should be static.

Reply:
That makes sense. Actually I copied this from the SNTP code, which probably also needs the static keyword.